support-frontend/assets/helpers/subscriptionsForms/submit.ts

```diff
@@ -221,10 +222,7 @@ function buildRegularPaymentRequest(
 		telephoneNumber: telephone,
 		product,
 		firstDeliveryDate: state.page.checkoutForm.product.startDate,
-		paymentFields: {
-			...paymentFields,
-			recaptchaToken,
-		},
+		paymentFields,
```

Comment:
It looks like the recaptcha token is only relevant here for certain payment methods (direct debit), is that right? (Whereas before it was always included in this data).

Reply:
Yes exactly 👍